core/src/main/java/org/elasticsearch/indices/query/IndicesQueriesRegistry.java

```diff
@@ -39,4 +41,11 @@ public IndicesQueriesRegistry(Settings settings, Map<String, QueryParser<?>> que
     public Map<String, QueryParser<?>> queryParsers() {
         return queryParsers;
     }
+    
+    /**
+     * Returns all registered sort parsers
+     */
+    public Map<String, SortBuilderTemp<?>> sortParsers() {
+        return sortParsers;
+    }
```

Comment:
This is unused so far, right? Because I can't see it beeing set in the constructor. If this is needed later on in fromXContent, then I have a similar problem with suggesters, maybe worth taking another look how this is wired. I have no answer here yet, but if it is unused for this PR, can we take it out again?